mountpoint-s3-client/tests/put_object_single.rs

```diff
@@ -805,3 +832,54 @@ async fn test_append_with_matching_checksum(checksum_algorithm: aws_sdk_s3::type
         .await
         .expect("append should succeed");
 }
+
+#[cfg(feature = "s3express_tests")]
+#[tokio::test]
+async fn test_append_crc64() {
```

Comment:
This test is for append where original object has not specified a checksum, so it was computed service-side. I think the name here should be specific for that.